src/middlewares/mail.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable import/prefer-default-export */
+import { verifyEmail } from '../services/mail/template/verifyEmail';
+import { emailVerifyToken } from '../utils/index';
+import Mail from '../services/mail/Mail';
+
+export const SendVerificationEmail = async (req, res, next) => {
+  let { email, firstName, lastName } = req.body;
+  email = email ? email.trim() : '';
+  firstName = firstName.trim();
```

Comment:
@zanio When I tested your `email/test` endpoint, I noticed you didn't validate the `req.body` and you didn't handle errors too. I noticed errors on my console.
<img width="1440" alt="Screenshot 2019-08-28 at 10 11 34 AM" src="https://user-images.githubusercontent.com/27569442/63842310-3e4ef380-c97c-11e9-8f9f-fd5a64fdb505.png">